src/issue/screens/issue.screen.js

```diff
@@ -214,14 +220,25 @@ class Issue extends Component {
     />;
 
   render() {
-    const { issue, comments, isPendingComments, navigation } = this.props;
+    const {
+      issue,
+      comments,
+      isPendingComments,
+      isPendingIssue,
+      navigation,
+    } = this.props;
 
     return (
       <ViewContainer>
-        {isPendingComments &&
-          <LoadingContainer animating={isPendingComments} center />}
+        {(isPendingComments || isPendingIssue) &&
+          <LoadingContainer
+            animating={isPendingComments || isPendingIssue}
+            center
+            text={'Retrieving issue'}
```

Comment:
Agreed - I really don't mind having text in all the loading icons but we can just leave it out to match the other screens for now.

I've been meaning to create a LoadingIssueScreen as a loading placeholder (just like the `LoadingRepositoryListScreen` for example) but I didn't have the time before first release and was like meh that can come later :P. Ideally we would have that instead of a spinner so I'm going to open an issue for that as a separate ticket.